include/nn/nex/berkeley.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <nn/types.h>
+
+namespace nn {
+namespace nex {
+
+namespace TransportProtocol {
+enum Type { unk1, unk2, unk3, unk4, unk5, unk6 };
+}
+
+namespace SocketDriver {
```

Comment:
When there are many lines inside a namespace, having an empty line after the opening brace and another empty line before the closing brace improves readability. Things look kinda cramped right now.